src/renderers/native/ReactNativeFeatureFlags.js

```diff
@@ -12,10 +12,8 @@
 
 'use strict';
 
-// Read from process.env in order to support Rollup flat bundles.
-// Jest test script will also write this value for Fiber tests.
 var ReactNativeFeatureFlags = {
-  useFiber: process.env.REACT_NATIVE_USE_FIBER,
+  useFiber: true,
```

Comment:
Note: we still have a shim that sets this to `false` by default, and the bundle code will ask for the shim. This file is only used for tests.